libstatistics_collector/test/topic_statistics_collector/test_received_message_period.cpp

```diff
@@ -35,9 +35,11 @@ constexpr const double kExpectedMaxMilliseconds{1000.0};
 constexpr const double kExpectedStandardDeviation{0.0};
 }  // namespace
 
+using ReceivedIntMessagePeriodCollector =
+  libstatistics_collector::topic_statistics_collector::ReceivedMessagePeriodCollector<int>;
```

Comment:
Suggestion: move this into the unnamed namespace

Reply:
Done.